mypy.ini

```diff
@@ -31,6 +31,7 @@ warn_unused_ignores = False
 
 files =
     torch,
+    torch/linalg,
```

Comment:
This should be covered... why is this necessary... did you just need to do this invalidate the cache?